velox/duckdb/conversion/DuckParser.h

```diff
@@ -32,7 +36,9 @@ namespace facebook::velox::duckdb {
 // are lower-cased, what prevents you to use functions and column names
 // containing upper case letters (e.g: "concatRow" will be parsed as
 // "concatrow").
-std::shared_ptr<const core::IExpr> parseExpr(const std::string& exprString);
+std::shared_ptr<const core::IExpr> parseExpr(
+    const std::string& exprString,
+    const parse::ParseOptions* options = nullptr);
```

Comment:
Any particular reason to pass options as raw pointer and not a reference? Do we need a default value?

Reply:
Passed a raw pointer here to allow a default value and forward declare `parse::ParseOptions`.
I will include the header and set a default.

Reply:
If your suggestion is to remove the default, we can do that. I don't think this is invoked in many places.

Reply:
It is easier to reason about the APIs when they don't use defaults. If a default is not necessary here, then let's remove it.